data/net/minecraft/server/level/DistanceManager.mapping

```diff
@@ -24,6 +24,9 @@ CLASS net/minecraft/server/level/DistanceManager
 		ARG 1 chunkPos
 	METHOD getNaturalSpawnChunkCount ()I
 		COMMENT Returns the number of chunks taken into account when calculating the mob cap
+	METHOD getTicketLevelAt (Lnet/minecraft/util/SortedArraySet;)I
+		COMMENT Gets the {@link net.minecraft.server.level.Ticket#getTicketLevel level} of the ticket.
```

Comment:
I think that should be a `{@linkplain}`, since the label text is intended to be non-monospaced.
```suggestion
		COMMENT Gets the {@linkplain net.minecraft.server.level.Ticket#getTicketLevel level} of the ticket.
```